mlflow/server/js/src/experiment-tracking/components/traces/TracesViewDeleteTraceModal.tsx

```diff
@@ -22,10 +23,11 @@ export const TracesViewDeleteTraceModal = ({
   const [errorMessage, setErrorMessage] = useState<string>('');
   const [isLoading, setIsLoading] = useState(false);
   const tracesToDelete = keys(pickBy(rowSelection, (value) => value));
+  const { experimentId } = useParams<{ experimentId: string }>();
```

Comment:
@hubertzub-db looks like the trace backend on databricks does not require this param, while the OSS implementation does. is this the right way to get the current experiment ID? it looks like that's the source of truth from further up

i was concerned about having multiple experiment IDs here, but it seems at the moment it's broken anyway:

<img width="1728" alt="Screenshot 2024-07-12 at 12 27 12 PM" src="https://github.com/user-attachments/assets/bb18b59a-9eab-44ba-9121-67b87d29ece9">

it seems to require a backend change (`URLSearchParams` condenses lists into comma-separated strings, so we need to parse this on the server), so i'll leave this to another PR

Reply:
> is this the right way to get the current experiment ID? it looks like that's the source of truth from further up

It's correct in the current context, but I think it would be better to pass `experimentId` (or `experimentIds`) from somewhere upstream (e.g. `<TracesView>` has it already) and drill it down to `<TracesViewDeleteTraceModal>`. After all, this component may be reusable in a different context in the future and shouldn't be dependent on particular URL shape.

Reply:
makes sense, i'll make the change!